tests/install/openqa_webui.pm

```diff
@@ -35,6 +35,10 @@ sub install_from_pkgs {
         assert_script_run 'systemctl enable apache2';
         assert_script_run 'systemctl restart apache2';
     }
+    if (get_var('FULL_OPENSUSE_TEST')) {
+        # avoid second git fetch after fetchneedles
+        assert_script_run q{if [ -e /etc/openqa/openqa.ini ]; then sed -i -e 's/#\[scm git\]/[scm git]\ngit_auto_clone = no\ngit_auto_update = no/' /etc/openqa/openqa.ini; else echo -e "[scm git]\ngit_auto_clone = no\ngit_auto_update = no" > /etc/openqa/openqa.ini.d/git.ini; fi};
```

Comment:
Maybe we can just assume that `/etc/openqa/openqa.ini.d` is supported.

Reply:
> Why is this in the `install_from_pkgs`.

That's the only place where we edit openqa.ini currently. Directly below this.

Reply:
> Maybe we can just assume that `/etc/openqa/openqa.ini.d` is supported.

I just used the same as below, so I'd say it would just look weird if we still have that logic below but not in the code I introduce.

Reply:
Dont want to be destructive but I think it make sense to take those openqa.ini out of that function, wrap them into another with a corresponding name and add it in the `run` function. it will look more clear and will keep the functions clean. wdyt?

Reply:
I'm not sure why you're all complaining about this line but did not complain about line 43 below

Reply:
Because line 43 was introduced when the new config parsing was just introduced so it had to support the old and new behavior. However, I don't think this is required anymore so always creating a drop-in config file should be enough.